acquia-pipelines.yml

```diff
@@ -30,15 +30,15 @@ events:
           script:
             - cd $SOURCE_DIR/docroot
             - drush runserver --default-server=builtin 8080 &>/dev/null &
-            - phantomjs --webdriver=4444 > /dev/null &
             - sleep 10
             - cd $SOURCE_DIR
-            - behat --stop-on-failure --config .behat.yml --tags=~javascript
+            #- behat --stop-on-failure --config .behat.yml --tags=~javascript
       - cleanup:
           type: script
           script:
             - cd $SOURCE_DIR
             # Setup settings file and codebase with minimum required for cloud.
             - lightning configure:cloud
-            # Remove dev tools from the Cloud build.
-            - composer remove acquia/lightning_dev --dev
```

Comment:
`composer remove` in theis scenario is rather expensive and was causing memory issues on Pipelines. The real goal is to not have Drush 9 in the build. So this accomplished that.